### DISCUSSION TITLE
Add Comments To Dictionary

### DISCUSSION THREAD

**Discussion by LeStarch** [nasa/fprime#1128] [Call for Comments]
Dictionary elements (command, command arguments, etc) support comments. However, when writing out the Ai.xml dictionary, these comments get stripped out.  It would be nice to carry these through into the dictionary.xml such that they can be used when reading just the dictionary.

Possible uses:
1. Contextualized help for commands and command arguments
2. Building documentation with expanded information

With the comments being stripped out, it is much harder to access this information should it be needed.

**Comment by metzdigital** [comment]
Totally agree! This would help end users understand what the command does and could save us developers a phone call / email / time. Developers are filling out the comment field but I feel at the moment its all going to waste. I would love to see something like this:
![image](https://user-images.githubusercontent.com/2592986/144097107-989555c0-c379-4244-9f61-78ac3785e505.png)

**Comment by LeStarch** [comment]
Turns out, it looks like this information is **supposed** to be plumbed into the description attribute in the dictionary.

**Reply by LeStarch** [reply]
I am going to close this CFC as it is not a discussion, but a bug!
